docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,411 @@
+{{% /admonition %}}
+-->
+
+## Correlations
+
+<!-- Jay Goodson -->
+
+_Available in public preview in all editions of Grafana._
+
+You can now bring context from multiple data sources into the Explore experience. Correlations is an extension of our existing data links functionality and now enables you to link from any data source to any other data source.
+
+Correlations enable you to seamlessly jump from one data source to another. You define relationships between your different data sources; when you're using Explore, simply click a button next to a related field in one data source and Grafana will run the corresponding query in the other data source.
+
+Correlations is currently in preview. As such, we recommended you only enable it on test or development instances, rather than in production environments.
+
+To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
+
+In subsequent releases, we’ll be refining and enhancing the user interface for Correlations, to provide a more streamlined user experience.
+
+## Scenes
+
+<!--Dominik Prokop & Natalia Bernarte -->
+
+_Experimental in all editions of Grafana._
+
+Scenes is a new front-end library by Grafana that empowers application engineers to effortlessly build stunning dashboard experiences right into their products. With Scenes, you can easily create apps that mirror the Grafana dashboarding experience, complete with template variable support, flexible layouts, dynamic panel rendering, and so much more.
+
+To try it out, go to the [@grafana/scenes](https://github.com/grafana/scenes) repository.
+
+To learn more, refer to the [Scenes documentation](https://grafana.github.io/scenes/).
+
+## Subfolders
+
+<!-- Zsofia K. -->
+
+_Available in public preview in all editions of Grafana._
+
+You can now create subfolders in Grafana to help you better organize your dashboards and alerts. This new feature allows you to create, read, update, and delete subfolders, making it easier to sort resources by business units, departments, and teams.
+
+You can also set up permissions using Role-Based Access Control (RBAC). Folder permissions cascade, being inherited from the parent folder, which simplifies access management.
+
+The ability to add subfolders is currently in preview. As such, it's recommended to enable it only on test or development instances, rather than in production environments.
+
+To get started creating subfolders, enable the `nestedFolders` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+In subsequent releases, we’ll be refining and enhancing the user interface for managing dashboards and folders, to provide a more streamlined user experience.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10.0-nested-folders-2.png" max-width="750px" caption="Subfolders in Grafana" >}}
+
```

Comment:
@zsofiakm 
I am not sure we can include this, unless we plan to backport https://github.com/grafana/grafana/pull/68124 to G10

Reply:
Updated the description so that it calls out that the feature is missing.